roles/cockpit-ui/tasks/main.yml

```diff
@@ -23,41 +23,38 @@
       tls_termination: passthrough
     register: registry_console_cockpit_kube
 
-  # XXX: Required for items still using command
-  - name: Create temp directory for kubeconfig
-    command: mktemp -d /tmp/openshift-ansible-XXXXXX
-    register: mktemp
-    changed_when: False
+  - name: Set registry-console params
+    set_fact:
+      l_registry_console_params:
+        OPENSHIFT_OAUTH_PROVIDER_URL: "{{ openshift.master.public_api_url }}"
+        REGISTRY_HOST: "{{ docker_registry_route.results[0].spec.host }}"
+        COCKPIT_KUBE_URL: "https://{{ registry_console_cockpit_kube.results.results[0].spec.host }}"
+        NODE_SELECTOR: "{{ openshift_hosted_registry_console_nodeselector | map_from_pairs }}"
```

Comment:
Default value for this variable appears to be a dictionary.  Does map_from_pairs handle this case?

Reply:
Defaults for both `openshift_hosted_registry_console_nodeselector` and `openshift_hosted_infra_selector` are strings.

Updated `hosts.example`